openshift/distro/ubi/9/Dockerfile

```diff
@@ -91,6 +91,11 @@ RUN cp -n openshift/3scale_backend.conf /etc/ \
  && touch /var/log/backend/3scale_backend{,_worker}.log \
  && chmod g+rw /var/log/backend/3scale_backend{,_worker}.log
 
+# Ensure safe defaults within the container for general case as well as to cover OpenShift specific case where the unprivileged user who runs the container is member of the root Linux group
+RUN chown -R 1001:0 /opt/ruby \
+    && chmod -R 750 /opt/ruby
```

Comment:
This is not directly related to the failure, but I found that this Dockerfile creates an image which is read-only for unprivileged users. `falcon hosts` creates a couple of temporary files when running, so it failed.

This is not a problem for alpha image because that Dockerfile changes the permissions to make the image writable for non-root users. I just copied that line from that Dockerfile to here.

Check the original: https://gitlab.cee.redhat.com/3scale/apisonator-midstream/-/blob/3scale-amp-2/distgit/containers/3scale-amp-backend/Dockerfile.in?ref_type=heads#L128-130